Examples/DataRepresentation/Containers/TreeContainer.cxx

```diff
@@ -52,7 +52,8 @@
 #include "itkTreeIteratorClone.h"
 // Software Guide : EndCodeSnippet
 
-int main(int, char* [])
+int
+main( int, char *[] )
```

Comment:
Do you really want to enforce having the return type of a function on a separate line? It's OK by me, I just have to get used to it. But if you do, you might as well consider using C++11 trailing return types everywhere:

    auto main( int, char *[] ) ->
    int

Reply:
This is mainly done to accommodate templated return types, such as `itk::SmartPointer<itk::Image<unsigned short int,3>>`. Having `int` and `void` on a line all by itself is quite annoying, but a price we will have to pay for consistency. I guess we will get used to it over time.

Reply:
It should be possible to manage long return types by tuning PenaltyReturnTypeOnItsOwnLine.